extmod/vfs_fat_file.c

```diff
@@ -277,7 +260,7 @@ const mp_obj_type_t mp_type_vfs_fat_textio = {
 STATIC mp_obj_t fatfs_builtin_open_self(mp_obj_t self_in, mp_obj_t path, mp_obj_t mode) {
```

Comment:
I think this entire function can now be merged with `file_open` above.

Reply:
Done.